components/localize-mixin.js

```diff
@@ -0,0 +1,11 @@
+import { LocalizeDynamicMixin } from '@brightspace-ui/core/mixins/localize-dynamic-mixin.js';
+
+export const FilterLocalizeMixin = superclass => class extends LocalizeDynamicMixin(superclass) {
```

Comment:
Minor: Seems like this is all langterms (not just filter langterms) so something like just `LocalizeMixin` would probably be better but the name doesn't _really_ matter....

Reply:
Agreed yeah... I didn't want to call it `LocalizeMixin` because that's what the "core" one is called... I guess `FacetFilterSortLocalizeMixin` is the closest but that's a mouthful.